astropy/units/quantity_helper/erfa.py

```diff
@@ -73,18 +97,250 @@ def helper_gd2gc(f, nounit, unit1, unit2, unit3):
                             .format(f.__name__))
 
 
+def helper_p2pv(f, unit1):
+    from astropy.units.si import s
+    if isinstance(unit1, StructuredUnit):
+        raise UnitTypeError("p vector unit cannot be a structured unit.")
+    return [None], StructuredUnit((unit1, unit1 / s))
+
+
+def helper_pv2p(f, unit1):
+    check_structured_unit(unit1, dt_pv)
+    return [None], unit1[0]
+
+
+def helper_pv2s(f, unit_pv):
+    from astropy.units.si import radian
+    check_structured_unit(unit_pv, dt_pv)
+    ang_unit = radian * unit_pv[1] / unit_pv[0]
```

Comment:
decompose? 
```suggestion
    ang_unit = (radian * unit_pv[1] / unit_pv[0]).decompose()
```

Reply:
I think it is fine to let the user do that, especially as `.decompose` is rather agressive. E.g.,
```
In [4]: (u.rad * u.AU / u.day / u.AU)
Out[4]: Unit("rad / d")

In [5]: (u.rad * u.AU / u.day / u.AU).decompose()
Out[5]: Unit("1.15741e-05 rad / s")
```